ui.frontend/src/view/FormFieldBase.js

```diff
@@ -332,25 +335,29 @@ class FormFieldBase extends FormField {
      * @private
      */
     #triggerEventOnGuideBridge(eventType, originalEventPayload) {
-        const formId = this.formContainer.getFormId();
-        const formTitle = this.formContainer.getFormTitle();
-        const panelName = this.#getPanelName();
-        const fieldName = this._model.name;
-        const fieldId = this._model.id;
-        const fieldQualifiedName = this._model.qualifiedName;
-        const eventPayload = {
-            formId,
-            formTitle,
-            fieldName,
-            fieldId,
-            panelName,
-            fieldQualifiedName,
-            ...(originalEventPayload?.prevValue !== undefined ? { prevText: originalEventPayload?.prevValue } : {}),
-            ...(originalEventPayload?.currentValue !== undefined ? { newText: originalEventPayload?.currentValue } : {}),
-            ...((typeof originalEventPayload === 'object' && originalEventPayload !== null) ? originalEventPayload : {})
-        };
-        const formContainerPath = this.formContainer.getPath();
-        window.guideBridge.trigger(eventType, eventPayload, formContainerPath);
+        try {
+            const formId = this.formContainer.getFormId();
+            const formTitle = this.formContainer.getFormTitle();
+            const panelName = this.#getPanelName();
+            const fieldName = this._model.name;
+            const fieldId = this._model.id;
+            const fieldQualifiedName = this._model.qualifiedName;
+            const eventPayload = {
+                formId,
+                formTitle,
+                fieldName,
+                fieldId,
+                panelName,
+                fieldQualifiedName,
+                ...(originalEventPayload?.prevValue !== undefined ? { prevText: originalEventPayload?.prevValue } : {}),
+                ...(originalEventPayload?.currentValue !== undefined ? { newText: originalEventPayload?.currentValue } : {}),
+                ...((typeof originalEventPayload === 'object' && originalEventPayload !== null) ? originalEventPayload : {})
+            };
+            const formContainerPath = this.formContainer.getPath();
+            window.guideBridge.trigger(eventType, eventPayload, formContainerPath);
+        } catch (e) {
+            console.log('some error occurred while triggering event on guideBridge', e);
```

Comment:
Why did you add a try catch here, ideally the code should break. Can you describe the use-case